tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.Datadog.Trace.PublicApiHasNotChanged.verified.txt

```diff
@@ -263,6 +263,7 @@ namespace Datadog.Trace.Configuration
         public bool AnalyticsEnabled { get; }
         public string? CustomSamplingRules { get; }
         public string? Environment { get; }
+        public bool ExperimentalAppsecStandaloneEnabled { get; }
```

Comment:
We shouldn't make this public, I think these are left over from a previous version?
```suggestion
```